test_app/custom_mutations.py

```diff
@@ -1,17 +0,0 @@
-import strawberry
```

Comment:
This PR currently removes `custom_mutations.py` and `custom_queries.py` entirely, but I could see it being helpful to leave the files there as templates and just removing the mutations/queries. LMK which is preferred!

Reply:
Yeah I think we could have an `examples` folder for that kind of thing?

Reply:
sgtm, I'll merge this one as-is for now and we can set up `examples` in a follow up 😄